lesson24/script.js

```diff
@@ -0,0 +1,199 @@
+//1.Подсчитать сумму всех чисел в заданном пользователем диапазоне.
+
+let firstNumber = prompt("Введите пожалуйста первое число:");
+let secondNumber = prompt("Введите пожалуйста Второе число:");
+let sum = 0;
+for (i = firstNumber; i <= secondNumber; i++) {
+  sum += firstNumber++;
+}
+alert(`Ваш ответ: ${sum}`);
+
+//2.Запросить 2 числа и найти только наибольший общий делитель.
+
+let firstNumber = prompt("Введите пожалуйста первое число:");
+let secondNumber = prompt("Введите пожалуйста Второе число:");
+let constant = 0;
+
+if (secondNumber > firstNumber) {
+  constant = secondNumber;
+} else {
+  constant = firstNumber;
+}
+for (i = constant; i >= 1; i--) {
+  if (secondNumber % i == 0 && firstNumber % i == 0) {
+    alert(`Наибольший общий делитель: ${i}`);
+    break;
+  }
+}
+
+// document.write(`Наибольший общий делитель: ${i}`);
+//Нашел такой вариант вывода ответа.Думаю в Примерах где ответ слишком большой и занимает много места ето норм сработалоб)
+
+//3.Запросить у пользователя число и вывести все делители этого числа.
+
+let number = prompt("Введите пожалуйста число:");

```

Comment:
лучше использовать `const`, если ничего позже не присваивается, и так везде 
```suggestion
const number = +prompt("Введите пожалуйста число:");

```